packages/story-editor/src/components/videoTrim/generateVideoStrip.js

```diff
@@ -17,31 +17,13 @@
 /**
  * External dependencies
  */
-import type { VideoResource } from '@googleforcreators/types';
+import {
+  getMediaSizePositionProps,
+  preloadVideo,
+  seekVideo,
+} from '@googleforcreators/media';
 
-/**
- * Internal dependencies
- */
-import getMediaSizePositionProps from './getMediaSizePositionProps';
-import preloadVideo from './preloadVideo';
-import seekVideo from './seekVideo';
-
-const CACHE: Map<string, string> = new Map();
-
-interface FlipSettings {
-  vertical: boolean;
-  horizontal: boolean;
-}
-
-// TODO(#12126): Use improved Element type from shared package.
-interface Element {
```

Comment:
Kinda sad having to remove the types again 😢

Reply:
Yes, that did hurt a bit. But they weren't that precise anyway 🤷. In the future we could move this to a `story-editor-util` library, that only the story editor depends on. It could have a ton of functionality that we'd want to typify but can't otherwise move too far out of the main package.

Reply:
It should be possible to just use TypeScript for individual files in `story-editor`, no need to create a separate package for that. Just need to adjust `"include": ["src/**/*"]` in `tsconfig.json` to list individual files. Great for migrating step by step. Or is there another benefit of having such a separate package?

Reply:
Oh, that sounds awesome. I thought it was an `all-or-nothing` deal. Let's explore that somewhere closer to the end of these migrations 😄